routes/api/crowbar/repocheck.js

```diff
@@ -1,14 +1,35 @@
 var express = require('express'),
-    router = express.Router();
+    router = express.Router(),
+    repo_correct = false;
 
 /* GET Admin Repo Checks. */
 router.get('/', function(req, res) {
-    res.status(200).json({
-        'SLES_12_SP2': true,
-        'SLES_12_SP2_Updates': true,
-        'SLES_OpenStack_Cloud_7': true,
-        'SLES_OpenStack_Cloud_7_Updates': true
-    });
+    var data = {
+        'os': {
+            'available': true,
+            'repos': {}
+        },
+        'openstack': {
+            'available': false,
+            'repos': {
+                'x86_64': {
+                    'missing': [
+                        'SUSE OpenStack Cloud 7',
```

Comment:
I think this should be updated to use same repository names listed in 
https://trello.com/c/oGniN5Lf/149-3-s49p19-32-admin-server-repo-checks-service-integration#comment-57f63b68212d25bfb1fd57af